gallery/src/pageConfig.ts

```diff
@@ -140,141 +140,151 @@ import NxStatefulCollapsibleRadioSelectPage
-    NxStatefulTreeViewRadioSelect: NxStatefulTreeViewRadioSelectPage,
-    NxVulnerabilityDetails: NxVulnerabilityDetailsPage
+  'Alerts and Indicators': {
+    'Alert': { content: NxAlertComponentsPage, type: 'react' },
+    'Stateful Alert': { content: NxStatefulAlertComponentsPage, type: 'react' },
+    'counter': { content: NxCounterPage, type: 'html' },
+    'Load Error': { content: NxLoadErrorPage, type: 'react' },
+    'Load Wrapper': { content: NxLoadWrapperPage, type: 'react' },
+    'Loading Spinner': { content: NxLoadingSpinnerPage, type: 'react' },
+    'Policy Violation Indicator': { content: NxPolicyViolationIndicatorPage, type: 'react' },
+    'Submit Mask': { content: NxSubmitMaskPage, type: 'react' },
+    'Stateful Submit Mask': { content: NxStatefulSubmitMaskPage, type: 'react' },
+    'Tag': { content: NxTagPage, type: 'react' },
+    'Threat Counter': { content: NxThreatCounterPage, type: 'react' },
+    'Small Threat Counter': { content: NxSmallThreatCounterPage, type: 'react' },
+    'Threat Indicator': { content: NxThreatIndicatorPage, type: 'react' },
+    'Threat Indicator Legend': { content: NxThreatIndicatorLegendPage, type: 'react' },
+    'threat-number': { content: NxThreatNumberPage, type: 'html' },
+    'Tooltip': { content: NxTooltipPage, type: 'react' },
+    'Overflow Tooltip': { content: NxOverflowTooltipPage, type: 'react' },
+    'System Notice': { content: NxSystemNoticePage, type: 'html' }
   },
-  'Guidelines': {
-    'Accessibility': AccessibilityPage,
-    'Additional Resources': AdditionalResourcePage,
-    'Contributing to the Gallery': ContributingPage,
-    'font-size and line-height': FontSizePage,
-    'Form Validation Guidelines': FormValidationPage,
-    'Styling Components': StylingComponentsPage
+  'Buttons and Dropdowns': {
+    'Back Button': { content: NxBackButtonPage, type: 'react' },
+    'Button': { content: NxButtonPage, type: 'react' },
+    'Segmented Button': { content: NxSegmentedButtonPage, type: 'react' },
+    'Stateful Segmented Button': { content: NxStatefulSegmentedButtonPage, type: 'react' },
+    'Close Button': { content: NxCloseButtonPage, type: 'react' },
+    'Dropdown': { content: NxDropdownPage, type: 'react' },
+    'Stateful Dropdown': { content: NxStatefulDropdownPage, type: 'react' },
+    'Icon Dropdown': { content: NxIconDropdownPage, type: 'react' },
+    'Stateful Icon Dropdown': { content: NxStatefulIconDropdownPage, type: 'react' },
+    'Pagination': { content: NxPaginationPage, type: 'react' },
+    'Indeterminate Pagination': { content: NxIndeterminatePaginationPage, type: 'react' },
+    'Search Dropdown': { content: NxSearchDropdownPage, type: 'react' },
+    'Stateful Search Dropdown': { content: NxStatefulSearchDropdownPage, type: 'react' },
+    'Text Link': { content: NxTextLinkPage, type: 'react' },
+    'Tooltip': { content: NxTooltipPage, type: 'react' },
+    'Overflow Tooltip': { content: NxOverflowTooltipPage, type: 'react' }
```

Comment:
These two are already in Alerts and Indicators and I think they are more correct there than here.

```suggestion
```